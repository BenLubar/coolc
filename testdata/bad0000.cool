class Main() extends ArrayAny(1) {}
