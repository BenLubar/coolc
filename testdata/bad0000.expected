testdata/bad0000.cool:1:22: cannot extend ArrayAny
